griptape/memory/structure/base_conversation_memory.py

```diff
@@ -10,18 +10,19 @@
 from griptape.mixins import SerializableMixin
 
 if TYPE_CHECKING:
-    from griptape.drivers import BaseConversationMemoryDriver
+    from griptape.drivers import BaseConversationMemoryDriver, BasePromptDriver
     from griptape.memory.structure import Run
-    from griptape.structures import Structure
 
 
 @define
 class BaseConversationMemory(SerializableMixin, ABC):
     driver: Optional[BaseConversationMemoryDriver] = field(
         default=Factory(lambda: Defaults.drivers_config.conversation_memory_driver), kw_only=True
     )
```

Comment:
Side conversation: should this (and other similar instances) be renamed to `conversation_memory_driver`?

Reply:
I prefer short names, BUT I also like consistency.

I vote for consistency